src/Sarif.UnitTests/Sarif.UnitTests.csproj

```diff
@@ -48,7 +48,10 @@
     <None Update="TestData\InsertOptionalDataVisitor\CoreTests_ContextCodeSnippets.sarif">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="TestData\InsertOptionalDataVisitor\CoreTests_Hashes+TextFiles+ComprehensiveRegionProperties+RegionSnippets+ContextCodeSnippets.sarif">
+    <None Update="TestData\InsertOptionalDataVisitor\CoreTests_FlattenedMessages.sarif">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
>CopyToOutputDirectory [](start = 7, length = 21)

Larry, would love to deploy your recommendation to provide a global copy reference here. The code you provided worked on the command-line but failed for me within the IDE however. So I continue the inferior pattern here. I don't know whether this is a bug in the IDE or if it relates to some other configuration gap...

Reply:
Ok, thanks for trying.

---
In reply to: [207575996](https://github.com/Microsoft/sarif-sdk/pull/984#discussion_r207575996) [](ancestors = 207575996)